core/src/test/java/uk/gov/gchq/koryphe/impl/function/IterableFlattenTest.java

```diff
@@ -50,7 +51,7 @@ public void shouldHandleNullInput() {
     public void shouldFlattenIterableNumbers() {
         // Given
         final IterableFlatten<Number> function = new IterableFlatten<>(new Sum());
-        final List<Number> input = Lists.newArrayList(1, 2, 3, 4, 5);
+        final List<Number> input = Lists.newArrayList(1, 2, 3, 4, null, 5);
```

Comment:
Instead of adding into this test, couldn't we have a new test for this? Such as 'shouldFlattenIterableNumbersWithNull'